package.json

```diff
@@ -4,21 +4,24 @@
   "description": "Simorgh",
   "main": "index.js",
   "scripts": {
+    "bbc-a11y": "bbc-a11y",
     "build": "razzle build",
     "cypress": "cypress run",
     "cypress:interactive": "cypress open",
     "cypress:storybook": "cypress run --project ./.storybook/.",
     "cypress:storybook:interactive": "cypress open --project ./.storybook/.",
     "data-validate": "node src/app/dataValidator/index.js",
     "dev": "razzle start",
-    "lighthouse": "lighthouse http://localhost:7080/article/id --output-path=./reports/report.html --view",
+    "lighthouse": "lighthouse http://localhost:7080/article/scenario-01 --output-path=./reports/report.html --view",
     "postmerge": "./scripts/dependencyUpdateCheck.sh",
     "precommit": "npm run test:lint",
     "prepush": "./scripts/upToDateWithLatest.sh && npm run test:unit",
     "speculate": "speculate",
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run test:unit && npm run data-validate",
+    "test:accessibility": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start bbc-a11y",
```

Comment:
Not in this PR, but we should move `(lsof -t -i:7080 | xargs kill)` to its own script - we are using it in multiple places now.